src/index.css

```diff
@@ -98,3 +191,272 @@ body {
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  text-decoration: none;
+}
+
+.CATEGORIES {
+  width: 6.563rem;
+  height: 1rem;
+  margin: 1.125rem 1.125rem 0.688rem 2.563rem;
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  text-decoration: none;
+}
+
+.oval {
+  display: flex;
+  width: 2.813rem;
+  height: 2.813rem;
+  border: solid 1px #e8e8e8;
+  border-radius: 50%;
+  background-color: var(--white);
+  cursor: pointer;
+}
+
+.user-icon {
+  margin: auto;
+  width: 1.68rem;
+  height: 1.68rem;
+  color: var(--azure);
+}
+
+.categorie {
+  padding-bottom: 0.5rem;
+}
+
+.author {
+  padding-bottom: 1.5rem;
+}
+
+.btn-container-book {
+  display: flex;
+  justify-content: space-between;
+}
+
+.btn-book-item {
+  position: relative;
+  width: 5rem;
+  height: 1.5rem;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  color: #4386bf;
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+  transition: color 0.3s, transform 0.3s;
+}
+
+.btn-book-item::before {
+  content: '';
+  position: absolute;
+  top: 100%;
+  left: 10%;
+  width: 0;
+  height: 1px;
+  background-color: var(--azure);
+  transition: width 0.3s;
+}
+
+.btn-container-book .btn-book-item::after {
+  content: '';
+  position: absolute;
+  width: 0;
+  height: 100%;
+  border: #e8e8e8 1px solid;
+  top: 0;
+  right: 0;
+}
+
+.btn-container-book .btn-book-item:last-child::after {
+  display: none;
+}
+
+.btn-book-item:hover {
+  color: var(--azure);
+}
+
+.btn-container-book .btn-book-item:first-child {
+  padding-left: 0;
+}
+
+.btn-book-item:hover::before {
+  width: 80%;
+}
+
+.progress-circle {
+  position: relative;
+  width: 80px;
+  height: 80px;
+  border-radius: 50%;
+  background-color: lightgray;
+  overflow: hidden;
+}
+
+.progress-circle::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+  background: linear-gradient(to left, var(--azure) 70%, lightgray 0);
+  transform-origin: left center;
+  transform: rotate(calc(360deg * var(--percent) / 100));
+  transition: transform 1.5s;
+}
+
+.progress-circle::after {
+  content: '';
+  position: absolute;
+  top: 8%;
+  left: 8%;
+
+  /* transform: translate(-50%, -50%); */
+  width: 84%;
+  height: 84%;
+  border-radius: 50%;
+  background: var(--white-two);
+  transform-origin: left center;
+  transform: rotate(calc(360deg * var(--percent) / 100));
+}
+
+.progress-circle[data-percent="15"]::before {
+  transform: rotate(360deg);
+}
+
+.current-chapter {
+  padding: 0.5rem 0;
+}
```

Comment:
- [ ] Please set `text-transform: uppercase` so that it looks just like the Zeplin design 👍 
### Zeplin Design
![image](https://github.com/jicamargo/bookstore/assets/46839548/66b45773-32d7-403b-b73f-5d737980d5fb)

### Your Design
![image](https://github.com/jicamargo/bookstore/assets/46839548/5c83f34a-00f4-4d25-8c8e-d9fbcae6b0b6)